package.json

```diff
@@ -72,31 +72,31 @@
     "gulp-filter": "^6.0.0",
     "gulp-if": "^3.0.0",
     "gulp-plumber": "^1.2.1",
-    "gulp-postcss": "^8.0.0",
+    "gulp-postcss": "^9.0.0",
     "gulp-rename": "^2.0.0",
     "gulp-sass": "^5.0.0",
     "gulp-sass-lint": "^1.4.0",
     "gulp-task-listing": "^1.1.0",
     "gulp-uglify": "^3.0.2",
     "html5shiv": "^3.7.3",
-    "jest": "^27.4.3",
+    "jest": "^29.7.0",
     "jest-axe": "^5.0.1",
-    "jest-puppeteer": "^6.0.2",
+    "jest-environment-jsdom": "^29.7.0",
+    "jest-puppeteer": "^9.0.2",
     "jest-serializer-html": "^7.0.0",
     "jquery": "1.12.4",
     "js-yaml": "^3.14.0",
     "map-stream": "0.0.7",
     "marked": "^4.0.10",
-    "node-sass": "^7.0.1",
+    "node-sass": "^9.0.0",
     "nodemon": "^2.0.4",
     "nunjucks": "^3.2.4",
     "oldie": "^1.3.0",
     "outdent": "^0.7.1",
     "path-webpack": "^0.0.3",
     "postcss": "^8.4.7",
-    "postcss-pseudo-classes": "0.2.0",
+    "postcss-pseudo-classes": "0.4.0",
     "postcss-scss": "^4.0.3",
-    "puppeteer": "^9.1.1",
```

Comment:
@oluoluoxenfree Reintroduce puppeteer into the package.json as per https://www.npmjs.com/package/jest-puppeteer?activeTab=readme